filters/openpolicyagent/openpolicyagent_test.go

```diff
@@ -63,22 +64,32 @@ func TestInterpolateTemplate(t *testing.T) {
 
 func TestLoadEnvoyMetadata(t *testing.T) {
 	cfg := &OpenPolicyAgentInstanceConfig{}
-	WithEnvoyMetadataBytes([]byte(`
+	_ = WithEnvoyMetadataBytes([]byte(`
 	{
 		"filter_metadata": {
 			"envoy.filters.http.header_to_metadata": {
 				"policy_type": "ingress"
+			},
+			"open_policy_agent" : {
+				"decision_id" : "3b567656-bf28-4a63-a4c4-14407fbd9544"
 			}
 		}
 	}
 	`))(cfg)
 
 	expectedBytes, err := protojson.Marshal(&ext_authz_v3_core.Metadata{
-		FilterMetadata: map[string]*_struct.Struct{
+		FilterMetadata: map[string]*pbstruct.Struct{
 			"envoy.filters.http.header_to_metadata": {
-				Fields: map[string]*_struct.Value{
+				Fields: map[string]*pbstruct.Value{
 					"policy_type": {
-						Kind: &_struct.Value_StringValue{StringValue: "ingress"},
+						Kind: &pbstruct.Value_StringValue{StringValue: "ingress"},
+					},
+				},
+			},
+			"open_policy_agent": {
```

Comment:
As this should always be dynamic, it feels a bit strange to include it here. Can this be moved?